### PR TITLE
add tutorial section and TOC links, update API Version info

### PR DESCRIPTION
**Pull request by dzucker-tab**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -206,7 +207,7 @@ same:  
   "created_at":"2018-11-15T17:14:45Z"
 
 }
-
+[Glitch](http://glitch.com)
```
Comment (shinchris) [inline README.md:210]: Was this addition intentional?

File: README.md [hunk 2]
```diff
@@ -217,11 +218,25 @@ same:  
...
+In this tutorial, you can learn how to use a [Glitch](http://glitch.com) project to integrate your webhook message into [Slack](https://slack.com). Using these techniques can help you avoid commonly seen HTTP 400 errors.
+
+- [Send a Slack notification when a workbook is created](https://github.com/tableau/datadev-hackathon/wiki/Send-a-Slack-notification-when-a-workbook-is-updated)
+
 ## <a id="endpoints"></a>Tableau Server REST API Endpoints for Webhooks  
 
 ### API Version  
 
-All REST API endpoints for the initial release of webhooks are under the 3.6 API version. The base URL for this API version is: `https://{{server}}/api/3.6/`.
+Tableau REST API endpoints for managing your webhooks are available in API version 3.6 API. Tshe base URL for this API version is: `https://{{server}}/api/{API_version_number}/`.
```
Comment (shinchris) [inline README.md:239]: Typo: 'Tshe', in the second sentence.

**Review by shinchris — CHANGES_REQUESTED** [review]
Thanks Dan!

File: README.md (hunk 2 above)
Comment (dzucker-tab) [reply README.md]: fixed, thanks

File: README.md (hunk 1 above)
Comment (dzucker-tab) [reply README.md]: Nope, fixed, thanks
